### PR TITLE
vArmour AppController Solution

### PR DESCRIPTION
**Pull request by mocean-v**
Fixes #

## Proposed Changes

Created the following

1. varmour-logo.svg
2. vArmour_AppController.csv
3. Connector_vArmour_AppController_CEF.json
4. vArmourAppControllerAppBlack.png
5. vArmourAppControllerAppWhite.png
6. vArmour_AppContoller_Workbook.json
7. vArmour_AppController_Connector.md

### CODE REVIEW

File: Logos/varmour-logo.svg [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (elforb) [inline Logos/varmour-logo.svg:1]: Logo is not sized properly, viewbox is much larger than content so logo appears tiny.

File: Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
+{
+    "id": "vArmourAC",
+    "title": " vArmour Application Controller",
+    "publisher": "vArmour",
+    "descriptionMarkdown": "Common Event Format (CEF) is an industry standard format on top of Syslog messages, used by many security vendors to allow event interoperability among different platforms. By connecting your CEF logs to Azure Sentinel, you can take advantage of search & correlation, alerting, & threat intelligence enrichment for each log.",
```
Comment (elforb) [inline Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json:5]: This description only talks about CEF datatype, but should cover meaningful explanation of what data the data connector enables bringing in, in what format and pointer to appliance documentation as applies

File: Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        },
+        {
+            "description" : "Top 10 Application Protocols matching violations",
+            "query": "CommonSecurityLog\n| where DeviceVendor == \"vArmour\"\n| where DeviceProduct == \"AC\"\n| where Activity == \"POLICY_VIOLATION\"\n| summarize count() by ApplicationProtocol\n| top 10 by count_\n"
+        }
+
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (CEF)",
```
Comment (elforb) [inline Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json:38]: Data type name does not represent provider/appliance

File: Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+                    "type": "CopyableLabel"
+                }
+            ]
+        },
+        {
+            "title": "4. Secure your machine ",
+            "description": "Make sure to configure the machine's security according to your organization's security policy\n\n\n[Learn more >](https://aka.ms/SecureCEF)"
+        }
+    ]
+}
```
Comment (elforb) [inline Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json:141]: No metadata object is present

File: Solutions/vArmour/Workbooks/vArmour_AppContoller_Workbook.json [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+            "styleSettings": {
+              "showBorder": true
+            }
+          }
+        ]
+      },
+      "name": "group - 3"
+    }
+  ],
+  "fallbackResourceIds": [
```
Comment (elforb) [inline Solutions/vArmour/Workbooks/vArmour_AppContoller_Workbook.json:183]: Workbook should not contain fallbackResourceIds

File: Logos/varmour-logo.svg (hunk 1 above)
Comment (v-jayakal) [inline Logos/varmour-logo.svg:1]: Please move the logo to solutions folder, it should be at "Azure-Sentinel/Solutions/vArmour/Data Connectors/Logo".

File: vArmour_AppController_Connector.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Connect your vArmour Application Controller to Azure Sentinel
```
Comment (v-jayakal) [inline vArmour_AppController_Connector.md:1]: Move this file to solutions, suggested paths "Azure-Sentinel/Solutions/vArmour" or "Azure-Sentinel/Solutions/vArmour/Data Connectors"

**Comment by v-jayakal** [comment]
@mocean-v , Can you please provide more images for workbook. Suggestion is to divide the each chart into a single image (instead of placing all of them in a single image).

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: Logos/varmour-logo.svg (hunk 1 above)
Comment (mocean-v) [reply Logos/varmour-logo.svg]: Relocated logo to  Azure-Sentinel/Solutions/vArmour/Data Connectors/Logo

File: vArmour_AppController_Connector.md (hunk 6 above)
Comment (mocean-v) [reply vArmour_AppController_Connector.md:1]: Relocated to "Azure-Sentinel/Solutions/vArmour/Data Connectors"

File: Solutions/vArmour/Workbooks/vArmour_AppContoller_Workbook.json (hunk 5 above)
Comment (mocean-v) [reply Solutions/vArmour/Workbooks/vArmour_AppContoller_Workbook.json]: Removed fallbackResourceIds

File: Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json (hunk 4 above)
Comment (v-jayakal) [reply Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json:141]: Will be updated later

File: Logos/varmour-logo.svg (hunk 1 above)
Comment (v-jayakal) [reply Logos/varmour-logo.svg]: Will be updated later

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**